core/utils/poly-look/src/visualisations/charts/bubble-charts/iconCluster/iconCluster.js

```diff
@@ -0,0 +1,84 @@
+import { BubbleCluster } from "../..";
+
+const bubblePadding = 15;
```

Comment:
This is triple as above. Maybe define globals like SMALL_BUBBLE_PADDING, HUMONGOUS_BUBBLE_PADDING and things like that?